egs/ami/s5/local/tfrnnlm/rnnlm_data_prep.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# This script prepares the data directory used for TensorFlow based RNNLM traiing
+# it prepares the following file in the output-directory
+# $dir/wordlist.rnn.final : wordlist for RNNLM
+# $dir/{train/valid} : the text files
+# $dir/unk.probs : this file is optional for rnnlm-rescoring.  If provided, the
+# probability for <OOS> would be porportionally distributed among all OOS words
+
+set -e
+
+train_text=data/ihm/train/text
+nwords=9999
+
+. path.sh
+. cmd.sh
+
+. utils/parse_options.sh
+
+if [ $# != 1 ]; then
+   echo "Usage: $0 <dest-dir>"
+   echo "For details of what the script does, see top of script file"
+   exit 1;
+fi
+
+dir=$1
+srcdir=data/local/dict
+
+mkdir -p $dir
+
+cat $srcdir/lexicon.txt | awk '{print $1}' | sort -u | grep -v -w '!SIL' > $dir/wordlist.all
+
+# Get training data with OOV words (w.r.t. our current vocab) replaced with <unk>.
```

Comment:
I just updated my original comment with backticks so that github doesn't treat the EOS marker as formatting and remove it.